src/copy_tree.rs

```diff
@@ -38,38 +40,71 @@ pub fn copy_tree<ST: ReadTree, DT: WriteTree>(
         // deleted or changed while this is running.
         ui::set_bytes_total(source.size()?.file_bytes);
     }
+
+    // Target selected for copy (subpaths from here and included)
+    let target = &options.restore_only;
+    let target_tree: Vec<&str> = target.split('/').collect();
+
     ui::set_progress_phase("Copying");
```

Comment:
Rather than doing this inline in the copy, I'd prefer this be some kind of decorator on the tree or on iter_entries. Then we could also have `conserve ls --only` which seems useful. But that might be a more complicated refactoring, so feel free to leave it as just a TODO.

That'd also give us a place to do a binary search to find the subtree.

Reply:
Interesting! Yeah needs quite a bit of refactoring. Leaving it for now or room for a new issue